packages/chips/chip-set/index.js

```diff
@@ -0,0 +1,67 @@
+import React, {Component} from 'react';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import {MDCChipSetFoundation} from '@material/chips';
+
+import Chip from '../chip';
+
+export default class ChipSet extends Component {
+
+  foundation_ = null;
+
+  constructor(props) {
+    super(props);
+    this.maxId = 0;
+    this.state = {
+      chips: props.labels.map((label) => {
+        return {
+          label,
+          id: this.maxId++,
```

Comment:
Same question as above. Where are we using this? Is it just to assign a `key` to the iterator? If so we can pass a second argument to `map` and use that. E.g.: in `render`, we could do something like
```
{this.props.labels.map(label, index) => (
  <div key={index}>
    {label}
  </div>
)}`
```

Further, if chip labels are unique, we can just use those.

Reply:
Thanks for taking a look, Alex! I set things up this way to anticipate adding/removing input chips. Originally, I thought the ChipSet could internally manage updating the list of chip labels to be rendered, including assigning IDs. After talking to @moog16 it sounds more reasonable for the end developer to manage the list of chip labels, and for ChipSet to just handle rendering whatever's passed into it. Made a bunch of changes, PTAL.